### ISSUE TITLE
duplicate blocks when duplicating page

### ISSUE THREAD

**Issue by hohl** [sheadawson/silverstripe-blocks#110]
Is there any way to also duplicate all blocks on a page when duplicating a page using them? I guess that might be possible by overriding `Page::onBeforeDuplicate` but I didn't get it working yet. Did anybody did something already?

**Comment by sheadawson** [comment]
Yes `onBeforeDuplicate` should do it. Happy to merge a PR if someone wants to submit
